### ISSUE TITLE
App doesn't start

### ISSUE THREAD

**Issue by wolkenschieber** [oliexdev/openScale#450]
After updating the app to version code 43/version number 2.0.2 the app doesn't start anymore. Following exception is thrown
```
Build version: 2.0.2 
Build date: 1980-01-01 00:00:00 
Current date: 2019-03-29 07:42:52 
Device: Motorola Moto G Play 
 
Stack trace:  
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.health.openscale/com.health.openscale.gui.MainActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setChecked(boolean)' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2678)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2743)
	at android.app.ActivityThread.-wrap12(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1490)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6165)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:888)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:778)
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setChecked(boolean)' on a null object reference
	at com.health.openscale.gui.fragments.OverviewFragment.onCreateView(OverviewFragment.java:160)
	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2440)
	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:885)
	at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1229)
	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1295)
	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:686)
	at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManagerImpl.java:2057)
	at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1847)
	at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1802)
	at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1709)
	at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManagerImpl.java:2609)
	at androidx.fragment.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManagerImpl.java:2565)
	at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:245)
	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:525)
	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:179)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1270)
	at android.app.Activity.performStart(Activity.java:6704)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2641)
	... 9 more

```

**Comment by steko** [comment]
Should be already fixed as per #446

**Comment by wolkenschieber** [comment]
Yes, looks like dup.
